Add information about avatar thumbnails (#32)
kasenvr/vircadia-docs-sphinx: docs/source/create/avatars/avatar-standards.md

Depending on the avatar's design, the collision hulls can be very large or small. This occurs because High Fidelity analyzes the shape of the avatar's torso (from hips to head) and tries to find the best shape that encloses the mesh. For example, if your avatar has large hips or perhaps a fully extended tail, High Fidelity thinks that the tip of the tail is part of your hips, and makes a very large collision hull.  To reduce the size of the collision hull, you can add skeleton joints to your avatar's tail. 

### Avatar Thumbnails

> The Avatar app will look for a thumbnail with the same URL as your avatar but a .jpg extension.

Reviewer: ```suggestion
The Avatar app will look for a thumbnail with a .jpg extension with the same name and at the same URL location as your avatar's.
```

> The Avatar App will look for a thumbnail that has a .jpg extension and has the otherwise same URL to your avatar.

Reviewer 2: ```suggestion
The Avatar app will look for a thumbnail with the same URL as your avatar but a .jpg extension.
```

> For example, if your avatar is hosted on `http://vircadia.dev/avatar.fbx`, then it will look for a thumbnail at `http://vircadia.dev/avatar.jpg`.

Reviewer: ```suggestion
For example, if your avatar is hosted at `https://vircadia.dev/avatar.fbx`, then it will look for a thumbnail at `https://vircadia.dev/avatar.jpg`.
```

> For example, if your avatar is hosted on `http://vircadia.dev/avatar.fbx`, then it will look for a thumbnail on `http://vircadia.dev/avatar.jpg`.

Reviewer 2: ```suggestion
For example, if your avatar is hosted on `http://vircadia.dev/avatar.fbx`, then it will look for a thumbnail at `http://vircadia.dev/avatar.jpg`.
```
